lib/ansible/modules/packaging/language/pip.py

```diff
@@ -101,7 +101,9 @@
         in the system and you want to run pip for the Python 3.3 installation.
         It cannot be specified together with the 'virtualenv' parameter (added in 2.1).
         By default, it will take the appropriate version for the python interpreter
-        use by ansible, e.g. pip3 on python 3, and pip2 or pip on python 2.
+        use by ansible, e.g. pip3 on python 3, and pip2 or pip on python 2. Note that
+        the setuptools requirement is for the Python interpreter used by ansible, not
```

Comment:
It may make sense to link a config option (`ansible_python_interpreter`) here.

Reply:
figured out how to add a ref link, but put it in the notes section so it can be rendered.